tad/repositories/exceptions.py

```diff
@@ -0,0 +1,8 @@
+from tad.core.exceptions import TADError
+
+
+class RepositoryError(TADError):
```

Comment:
It is better to have one place for all exceptions. please use the core/exception file

Reply:
Okidoki.